js/localization.d.ts

```diff
@@ -66,15 +66,15 @@ export function locale(): string;
 
 /**
  * @docid localization.locale
- * @publicName locale(locale)
+ * @publicName locale(localeToSet)
  * @param1 locale:string
  * @static
  * @module localization
  * @export locale
  * @namespace DevExpress.localization
  * @public
  */
-export function locale(locale: string): void;
+export function locale(localeToSet: string): void;
```

Comment:
This change affects topcis generation
```suggestion
export function locale(locale: string): void;
```